spec/system/task_spec.rb

```diff
@@ -1,24 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+
   describe 'Task一覧' do
     context '正常系' do
+      let(:project) { create(:project) }
+      let(:task) { create(:task, project_id: project.id) }
```

Comment:
`association`の定義を使うと`project_id`の指定が不要になりますね。
```suggestion
      let(:task) { create(:task) }
```

Reply:
2513b0ea1bbb0ea749cdbd90f8a4f02425ef7112　で修正しました。
traitの中に記載していたassociationを外に出し、project_id: project.idを削除しました。